service/matching/matching_engine.go

```diff
@@ -418,20 +417,16 @@ func (e *matchingEngineImpl) AddActivityTask(
 		taskInfo:      taskInfo,
 		source:        addRequest.GetSource(),
 		forwardedFrom: addRequest.GetForwardedSource(),
-		baseTqm:       tqm,
 	})
 }
 
 func (e *matchingEngineImpl) DispatchSpooledTask(
```

Comment:
this whole function isn't needed anymore, is it?

Reply:
I think as an interface it makes sense, to keep `matcher` private